tests/test_objects.py

```diff
@@ -179,8 +179,8 @@ def test_GridFileObjects(self):
         """ Tests gridfs objects """
         conn = yield txmongo.MongoConnection(mongo_host, mongo_port)
         db = conn.test
-        db.fs.files.remove({})  # drop all objects there first
-        db.fs.chunks.remove({})
+        yield db.fs.files.remove({})  # drop all objects there first
+        yield db.fs.chunks.remove({})
```

Comment:
I suggest using `yield defer.gatherResults(db.fs.files.remove({}), db.fs.chunks.remove({}))` here to not limit concurrency